### PR TITLE
Add new parameters

### PR DESCRIPTION
**Pull request by domcleal**
Some new parameters that were added to the smart proxy between 1.8 and 1.9.  The only I haven't done yet is https://github.com/theforeman/smart-proxy/commit/75f587114cee27f084839f7e0d2cde013b986b7f, which probably needs more work due to sudoers rules.

### CODE REVIEW

File: templates/settings.yml.erb [hunk 1]
```diff
@@ -68,7 +68,9 @@
 # Uncomment and modify if you want to change the default pid file '/var/run/foreman-proxy/foreman-proxy.pid'
 #:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
 
-# HTTP ports configuration
+# host and ports configuration
+# host to bind ports to (possible values: *, localhost, 0.0.0.0)
```
Comment (ekohl) [inline templates/settings.yml.erb]: I'd leave ports out: ```host to bind on```. Pretty sure you can also include ```::``` for IPv6 and maybe include an example IP?

File: manifests/params.pp [hunk 2]
```diff
@@ -14,6 +14,7 @@
   $plugin_version = 'installed'
 
   # variables
+  $bind_host = '*'
```
Comment (ekohl) [inline manifests/params.pp]: Does * bind on IPv6 as well?

File: templates/settings.yml.erb (hunk 1 above)
Comment (domcleal) [inline templates/settings.yml.erb]: Yeah, I copied and pasted from the example config file, I agree it's not the best explanation.  Shall I improve this version?

File: manifests/params.pp (hunk 2 above)
Comment (domcleal) [inline manifests/params.pp]: It seems to for me.

Comment (domcleal) [inline manifests/params.pp]: In fact, I think we deliberately chose '*' as the default for that reason.. 0.0.0.0 was proposed originally.

Comment (ekohl) [inline manifests/params.pp]: I recall ```0.0.0.0``` used to be the default so this is an improvement :)

File: templates/settings.yml.erb (hunk 1 above)
Comment (ekohl) [inline templates/settings.yml.erb]: Maybe both?

**Comment by mmoll** [comment]
I'm :+1:

File: templates/settings.yml.erb (hunk 1 above)
Comment (mmoll) [inline templates/settings.yml.erb]: or ````IPs to bind on```? I'm OK either way :)

Comment (domcleal) [inline templates/settings.yml.erb]: Updated, should read better now.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**